localnet/poktrolld/config/supplier1_stake_config.yaml

```diff
@@ -15,5 +15,5 @@ services:
         rpc_type: JSON_RPC
   - service_id: ollama
     endpoints:
-      - publicly_exposed_url: http://relayminer1:8545
+      - publicly_exposed_url: http://relayminer2:8545
```

Comment:
Not sure if this is intended to be a working (non-kubernetes) example, so maybe it should remain `relayminer1`